src/pages/index.tsx

```diff
@@ -5,7 +5,7 @@ const inter = Inter({ subsets: ["latin"] });
 export default function Home() {
   return (
     <main className="bg-primary-color min-h-screen">
-      {/* <h1>amazing biztech app</h1> */}
+      <h1>amazing biztech app</h1>
```

Comment:
nit: GitHub says there's going to be a "merge conflict" on index.tsx because of this change lol